src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs

```diff
@@ -54,25 +54,29 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
         {
             if (e.PropertyName == nameof(WatchViewModel.IsCollection))
             {
-                if (_vm.IsCollection)
+                // // The WatchTree controll will resize only if its role is a WatchNode (starts with an specific height), otherwise it won't resize (Bubble role).
+                if (!Double.IsNaN(this.Height))
```

Comment:
Interesting, this looks a bit hacky but I could not find a property to indicate if current state is for preview bubble or watch node